test/testspacesmigration/migrationpayload.json

```diff
@@ -0,0 +1,19 @@
+{
+    "folders": {
+        "/": {
```

Comment:
In difference to the first proposal in https://github.com/owncloud/ocis/issues/3574#issuecomment-1110992758 each folder has an object and not an array.